demos-europe/demosplan-core
# feature (refs t33577): additional import features

**Ticket:** https://yaits.demos-deutschland.de/T33577

1. Include the count of skipped statements in the result flash message.
2. Automatically create planning document category if it does not exist yet in the procedure (WIP)

### How to review/test

Review code.

After implementing `findOrCreatePlanningCategory` test the import locally or on a server. I (cdr) did not yet test any of this.

Check translation messages and ideally add singular/plural support

### Linked PRs
#### depends on :arrow_up::
https://github.com/demos-europe/demosplan-core/pull/2331



### Tasks (optional)
A list of all related tasks that need to be done before this can be merged.

- [x] implement `\demosplan\DemosPlanCoreBundle\Logic\Import\Statement\StatementFromRowBuilder::findOrCreatePlanningCategory`: when creating planning document category entities ("Elements") a category type ("category") is needed. The `findOrCreatePlanningCategory` method has access to some information (see docblock) to guess the correct string. Implementing this method requires some deeper understanding of planning document categories, please consult other developers as needed.

### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
